app/models/metric/capture.rb

```diff
@@ -234,4 +238,5 @@ def self.minutes_ago(value)
       value.to_i_with_method.seconds.ago.utc
     end
   end
+  private_class_method :minutes_ago
```

Comment:
Thanks, will fix.  I could see it both ways.  Generally I've seen singleton class `class << self` notation elsewhere such as the rails codebase and have both positive/negative opinions for the reasons you mentioned.